docs/how-ubuntu-is-made/concepts/glossary.md

```diff
@@ -297,7 +297,9 @@ Circle of Friends
     ```
 
 Closed Source Software
-    *Work in Progress*
+    Software that does not allow public access to its source code. In Ubuntu, closed source software can typically be found in the {term}`multiverse` and {term}`restricted` components which include third party applications and proprietary drivers.
```

Comment:
```suggestion
    Software that does not allow public access to its source code. In Ubuntu, closed source software is found in the {term}`multiverse` and {term}`restricted` components which include third party applications and proprietary drivers.
```

Reply:
closed source software is *only* found in `restricted` and `multiverse` -- by contrast, the other two components, `main` and `universe` contain all the open source software packages.